common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,303 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_primitive = yes
+						is_country_type = primitive
+					}
+				}
+			}
+			energy = 500
+			food = 30
+			alloys = 300
+		}
+		cost = {
+			alloys = 100
+		}
+		produces = {
+			unity = 1
+		}
+		upkeep = {
+			food = 2
+			alloys = 2
+			energy = 2
+			minerals = 2
+		}
+	}
+
+	potential = {
+		owner = { has_country_flag = hfe_varelse_primitive_0 }
+		owner = {
+			OR = {
+				has_civic = hfe_civic_high_mechanicus
+				has_country_flag = high_mechanicus_retained
+				is_spiritualist = yes
+			}
+			NOT = {
```

Comment:
NOT = NOR

Reply:
Its kind of mod's style guidelines

Reply:
I am a NOT man, and I will always use NOT unless I am forced to do otherwise. I have written or changed seventeen thousand lines of code, and every one of them uses NOT for that reason.

I've led an open rebellion against NOR/NAND since the two switches were created. I cannot comprehend NAND at a quick glance. NOT and NOT AND are easier to read for people that are not as good at Mod making as you. I'm not an expert at this by any means. Also, NOR is s stupid because unlike NAND, the NOR command is a perfect duplicate of NOT. For someone that doesn't like redundancy and duplicity, why you think NOR is a good thing is beyond me.